src/main/java/net/sf/jabref/migrations/PreferencesMigrations.java

```diff
@@ -79,4 +88,43 @@ public static void upgradeSortOrder() {
         }
     }
 
+    /**
+     * Migrate LabelPattern configuration from version 3.3-3.5 to new BibtexKeyPatterns
```

Comment:
Minor comment: Could we also upgrade from <3.3? We still have users coming from JabRef 2.9.2.